detox/test/e2e/12.animations.test.js

```diff
@@ -9,7 +9,7 @@ describe('React-Native Animations', () => {
       let loopSwitch = element(by.id('UniqueId_AnimationsScreen_enableLoop'));
       await loopSwitch.tap();
       if (device.getPlatform() === 'ios') {
-        await expect(loopSwitch).toHaveValue('1');
+        await expect(loopSwitch).toHaveToggleValue('1');
```

Comment:
I think we should add a warning whenever `.toHaveValue()` is used over a `UISwitch`.

Reply:
Maybe, not sure, we can talk about that. AFAIK we never did it with other components (CMIIW)
It's a specific component behaviour that was changed over RN versions.. 🤷🏼‍♂️ 
It was really the accessibility value of the original component before